src/ocean/text/util/StringC.d

```diff
@@ -89,7 +89,9 @@ class StringC
     {
         if (str.length && !!str[$ - 1])
         {
+            enableStomping(str);
             str ~= StringC.Term;
```

Comment:
Yes, correct. I did found what's wrong with my code, but I wanted to put this `enableStomping` _before_ the call, but that's unnecessary, so I'll just close this PR.